ipalib/config.py

```diff
@@ -44,6 +44,7 @@
 from ipalib.constants import CONFIG_SECTION
 from ipalib.constants import OVERRIDE_ERROR, SET_ERROR, DEL_ERROR
 from ipalib import errors
+from ipaplatform.tasks import tasks
```

Comment:
Importing from `ipaplatform` to `ipalib` is forbidden, this will have to be worked around.